include/data/mart_items.h

```diff
@@ -0,0 +1,348 @@
+#include "constants/items.h"
+#include "constants/seals.h"
+
+typedef struct {
+    u16 itemID;
+    u16 unk_02;
```

Comment:
Is this for the badge-based shop levels?

Reply:
Yes, I forgot to update this